README.md

```diff
@@ -39,19 +55,36 @@ You can now build the PureScript source code with:
 npm run build
 ```
 
-You can produce a bundled JS file you can run in the browser with:
+### Launching the App
+
+You can launch your app in the browser with:
 
 ```sh
-# An alias for `spago bundle-app --to dist/app.js`
-npm run bundle
+# An alias for `parcel dev/index.html --out-dir dev-dist --open`
+npm run serve
 ```
 
-This deposits a bundled JS file named `app.js` in the `dist` directory. You can view your running Halogen app by opening the `dist/index.html` file.
+### Development Cycle
 
-Alternatively, if you use an editor that supports `purs ide` or if you are running [`pscid`](https://github.com/kRITZCREEK/pscid), then you can get near-instant builds of the app while you work:
+If you're using an [editor](https://github.com/purescript/documentation/blob/master/ecosystem/Editor-and-tool-support.md#editors) that supports [`purs ide`](https://github.com/purescript/purescript/tree/master/psc-ide) or are running [`pscid`](https://github.com/kRITZCREEK/pscid), you simply need to keep the previous `npm run serve` command running in a terminal. Any save to a file will trigger an incremental recompilation, rebundle, and web page refresh, so you can immediately see your changes.
 
-```sh
-npm run bundle:watch
+If your workflow does not support automatic recompilation, or if you add, remove, or modify module names, then you will need to manually re-run `npm run build`.
+
+:warning: `purs ide` only rebuilds one module at a time, so sometimes the bundle will end up in an inconsistent state, resulting in runtime errors. This occurs when a change is made in one module that breaks other modules that depend on it. The solution is to run a full build (`npm run build`) when a change like this is made, as the compiler will force you to resolve those errors.
```

Comment:
This was a warning specific to the previous "fast watch" command with Webpack. I'm not sure whether or not it is true with this setup (it could be -- not sure!)

Reply:
This seems to be true for all "watching" bundlers.

I left this in to minimize changes from what garyb wrote in the original template, since it has some good specific info, but maybe all the beginner cares about is:

> If your workflow does not support automatic recompilation, then you will need to manually re-run `npm run build`. Even with automatic recompilation, a manual rebuild is occasionally required, such as when you add, remove, or modify module names, or notice any other unexpected behavior.

Reply:
> maybe all the beginner cares about is:

I like this formulation. Mind updating the text to be this?